BitwardenShared/UI/Auth/Landing/LandingProcessor.swift

```diff
@@ -224,9 +233,6 @@ extension LandingProcessor: RegionDelegate {
         guard !urls.isEmpty else { return }
         await services.environmentService.setPreAuthURLs(urls: urls)
         state.region = region
-        // After setting a new region, feature flags need to be reloaded
-        Task {
-            await loadFeatureFlag()
-        }
+        await refreshConfig()
```

Comment:
⚠️ @phil-livefront This introduces several issues on slow internet connection:
1. On [perform](https://github.com/bitwarden/ios/blob/main/BitwardenShared/UI/Auth/Landing/LandingProcessor.swift#L67) `.appeared` It won't call  `refreshProfileState` because it's going to await until `refreshConfig` finishes.
2. When setting a Self-host environment it won't close the Self-host settings view until `refreshConfig` finishes given that it's called in [func didSaveEnvironment(urls: EnvironmentUrlData) async](https://github.com/bitwarden/ios/blob/main/BitwardenShared/UI/Auth/Landing/LandingProcessor.swift#L208)

I'd go back to wrap the config refresh call in a `Task` so we don't have these issues and if we **really need** to wait for the config for some thing, we'd need to use some other approach, but given this is only for feature flags I'm not that concerned that is not loaded correctly on low internet connection as we have fallback flows for `emailVerificationFeatureFlag`.